### PR TITLE
Add tweak for WorkQueueManager

### PR DESCRIPTION
**Pull request by todor-ivanov**
Fixes https://github.com/dmwm/WMCore/issues/11986

With this change we add the forgotten tweaks for testbed agents to use Producion Rucio instead of integration.

### CODE REVIEW

File: docker/pypi/wmagent/init.sh [hunk 1]
```diff
@@ -415,6 +421,10 @@ agent_tweakconfig() {
             GLOBAL_DBS_URL=https://cmsweb-testbed.cern.ch/dbs/int/global/DBSReader
             sed -i "s+DBSInterface.globalDBSUrl = 'https://cmsweb.cern.ch/dbs/prod/global/DBSReader'+DBSInterface.globalDBSUrl = '$GLOBAL_DBS_URL'+" $WMA_CONFIG_DIR/config.py
             sed -i "s+DBSInterface.DBSUrl = 'https://cmsweb.cern.ch/dbs/prod/global/DBSReader'+DBSInterface.DBSUrl = '$GLOBAL_DBS_URL'+" $WMA_CONFIG_DIR/config.py
+            rucio_host=http://cms-rucio.cern.ch
+            rucio_auth=https://cms-rucio-auth.cern.ch
+            sed -i "s+WorkQueueManager.rucioUrl =.*+WorkQueueManager.rucioUrl+WorkQueueManager.rucioUrl = $rucio_host'+" $WMA_CONFIG_DIR/config.py
```
Comment (amaltaro) [inline docker/pypi/wmagent/init.sh:426]: You have one item extra in this sed command, in addition to be missing a single quote. The command you wanted is likely: `sed -i "s+WorkQueueManager.rucioUrl =.*+WorkQueueManager.rucioUrl = '$rucio_host'+" $WMA_CONFIG_DIR/config.py`

But please make sure to test these out

File: docker/pypi/wmagent/init.sh [hunk 2]
```diff
@@ -415,6 +421,10 @@ agent_tweakconfig() {
...
+            sed -i "s+WorkQueueManager.rucioAuthUrl =.*+WorkQueueManager.rucioUrl+WorkQueueManager.rucioAuthUrl = $rucio_auth'+" $WMA_CONFIG_DIR/config.py
```
Comment (amaltaro) [inline docker/pypi/wmagent/init.sh:427]: Similar comment here.

**Review by amaltaro — CHANGES_REQUESTED** [review]
Todor, please find comments/requests along the code.

File: docker/pypi/wmagent/init.sh (hunk 1 above)
Comment (todor-ivanov) [reply docker/pypi/wmagent/init.sh]: thanks. fixed

File: docker/pypi/wmagent/init.sh (hunk 2 above)
Comment (todor-ivanov) [reply docker/pypi/wmagent/init.sh]: fixed

File: docker/pypi/wmagent/init.sh [hunk 3]
```diff
@@ -415,6 +421,10 @@ agent_tweakconfig() {
             GLOBAL_DBS_URL=https://cmsweb-testbed.cern.ch/dbs/int/global/DBSReader
             sed -i "s+DBSInterface.globalDBSUrl = 'https://cmsweb.cern.ch/dbs/prod/global/DBSReader'+DBSInterface.globalDBSUrl = '$GLOBAL_DBS_URL'+" $WMA_CONFIG_DIR/config.py
             sed -i "s+DBSInterface.DBSUrl = 'https://cmsweb.cern.ch/dbs/prod/global/DBSReader'+DBSInterface.DBSUrl = '$GLOBAL_DBS_URL'+" $WMA_CONFIG_DIR/config.py
+            rucio_host=http://cms-rucio.cern.ch
+            rucio_auth=https://cms-rucio-auth.cern.ch
+            sed -i "s+WorkQueueManager.rucioUrl = .*+WorkQueueManager.rucioUrl = $rucio_host+" $WMA_CONFIG_DIR/config.py
```
Comment (amaltaro) [inline docker/pypi/wmagent/init.sh:426]: This will break the configuration file, as you ditched out the single quotes making the url string.

**Review by amaltaro — CHANGES_REQUESTED** [review]

File: docker/pypi/wmagent/init.sh (hunk 3 above)
Comment (todor-ivanov) [reply docker/pypi/wmagent/init.sh]: fixed
Here is the output of applying the lines to the config file:
```
(WMAgent-2.3.4rc3) [cmst1@vocms0260:current]$ sed -i "s+WorkQueueManager.rucioAuthUrl = .*+WorkQueueManager.rucioAuthUrl = '$rucio_auth'+" $WMA_CONFIG_DIR/config.py

(WMAgent-2.3.4rc3) [cmst1@vocms0260:current]$ sed -i "s+WorkQueueManager.rucioUrl = .*+WorkQueueManager.rucioUrl = '$rucio_host'+" $WMA_CONFIG_DIR/config.py

(WMAgent-2.3.4rc3) [cmst1@vocms0260:current]$ grep cms-rucio $WMA_CONFIG_DIR/config.py |grep WorkQueueManager
config.WorkQueueManager.rucioUrl = 'http://cms-rucio.cern.ch'
config.WorkQueueManager.rucioAuthUrl = 'http://cms-rucio-auth.cern.ch'

```

Comment (amaltaro) [reply docker/pypi/wmagent/init.sh]: Wonderful, thanks Todor!

**Review by amaltaro — APPROVED** [review]
Looks good to me, thanks Todor!

**Pull request merged**